chess2.cpp

```diff
@@ -0,0 +1,882 @@
+#include "chess2.h"
+#include<iostream>
+
+int Chess2::perform(str Move) {
```

Comment:
Need code comment on what does `Move` represent... it  is might not be very clear what is expecteed inside `Move`

Reply:
naming convention to be followed..